cmd/gateway/main.go

```diff
@@ -78,8 +80,13 @@ func configureConnectionRegistrar(cfg *config.Config, localCM c.ConnectionRegist
 }
 
 func main() {
-	var wsAddr = flag.String("wsAddr", ":8080", "Hostname:port of the websocket server")
-	var mgmtAddr = flag.String("mgmtAddr", ":9090", "Hostname:port of the management server")
+	var wsAddr = *flag.String("wsAddr", ":8080", "Hostname:port of the websocket server")
+	var mgmtAddr = *flag.String("mgmtAddr", ":9090", "Hostname:port of the management server")
+
+	if os.Getenv("CLOWDER_ENABLED") == "true" {
```

Comment:
Please consider hiding this logic behind a method:
https://github.com/RedHatInsights/app-common-go/issues/10